forge/db/models/Project.js

```diff
@@ -185,6 +212,13 @@ module.exports = {
                 }
             },
             static: {
+                isNameUsed: async (name) => {
+                    const safeName = name?.toLowerCase()
+                    const found = await this.findOne({
+                        where: { safeName: safeName }
+                    })
+                    return !!found
```

Comment:
```suggestion
                    const count = await this.count({
                        where: { safeName: safeName }
                    })
                    return count !== 0
```

Reply:
Using `count` is a more optimal choice (as per the original code we had that checked for duplicates) - it removes the need to retrieve the whole database object